shopinvader_api_customer/routers/customer.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from typing import Annotated
+
+from fastapi import APIRouter, Depends
+
+from odoo.addons.base.models.res_partner import Partner as ResPartner
+from odoo.addons.fastapi.dependencies import authenticated_partner
+
+from ..schemas.customer import Customer
+
+# create a router
+customer_router = APIRouter(tags=["customer"])
+
+
+@customer_router.get("/customer", response_model=Customer)
+def get_customer_data(
+    partner: Annotated[ResPartner, Depends(authenticated_partner)],
+) -> Customer:
+    """
+    Get customer data of authenticated user
+    """
+    return Customer.from_res_partner(partner)
+
+
+@customer_router.post(
+    "/customer/update",
```

Comment:
No need of specifying the "update"

```suggestion
    "/customer",
```

Reply:
Done